### PR TITLE
Fixed maximize issue and minor ui improvements

### PR DESCRIPTION
**Pull request by st3no**
- Fixed maximize issue (on Linux OS) #37 
- Fixed about dialog #35
- Customized JFXDecorator
- Renamed MyTab 
- Made the button to close the tab nicer
- Refactor of color logic

### CODE REVIEW

**Comment by mayuso** [comment]
Sorry for the delay. I just wanted to let you know that I've seen this, I am just very busy lately.

I'll review and merge if everything is ok ASAP.

**Comment by mayuso** [comment]
Tested. It definitely has improved a lot. Thank you. 

It has a little mistake in the save window/pop-up but it's not a serious problem (It's just that the color doesn't load properly), I can fix it in the develop branch.

**Comment by mayuso** [comment]
With these changes I am feeling pretty good to do a 1.1 release. I just need to fix some little things and I might just release it. 

This month almost certainly.

**Comment by st3no** [comment]
I'm glad that it goes (almost) all right!

As for the "**save window**", I was already aware of the bug but the idea was to completely replace the window with a dialog (as done for the section about), it seems to me logically more correct and certainly will improve from the graphical point of view.

What do you think about it ?? :)

**Comment by mayuso** [comment]
**TL;DR:** Great idea, a lot of advantages without any disadvantage (IMO).

Sounds good to me, that way we would also fix #25 automatically.

Which is also one of the things i've been wanting to do for a long time, but never found a "clean" way to do. This way that you propose is a lot better than adjusting the pop-up using the current JMarkPad position and size. 

The only thing worth mentioning is that those dialogs can't be moved, which I don't see as a problem,  specially is we make all of them the same.

**Comment by st3no** [comment]
#25 will no longer be a problem!

For the time being I only work on the popup, after finishing it, you can make a last revision and then it will be ready for the release of 1.1 :)
